geomesa-tools/bin/geomesa-kafka

```diff
@@ -0,0 +1,251 @@
+#! /usr/bin/env bash
+#
+# Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0 which
+# accompanies this distribution and is available at
+# http://www.opensource.org/licenses/apache2.0.php.
+#
+
+setGeoHome ()
```

Comment:
Yeah, these bash scripts need some maintenance... I may leave it for a future PR though, as I don't know what's ideal.